LATEST

```diff
@@ -1,6 +1,6 @@
 d96b7bdc93379fd60f7292bf9e0b50e9573a566a 2.1.0-snapshot.20220223.9363.0.d96b7bdc SPLIT_RELEASE
 4038d0a7e3694305ba477cd9a4268400c39c20cd 2.0.0-snapshot.20220127.9042.0.4038d0a7
-b7fc9f5731aae9fdde364e6745dfb47ee1e10030 2.0.0-snapshot.20220209.9212.0.b7fc9f57 SPLIT_RELEASE
+b291a57e5c1c8fbed19dc1ffc3a7bce1bd121e04 2.0.0-snapshot.20220225.9368.0.b291a57e
```

Comment:
It still needs to be a SPLIT_RELEASE.

Reply:
fixed

Reply:
clearly shouldn’t be working at this point

Reply:
Maybe also worth adding a check to disallow non-split releases for newer versions.